src/cases/NgRestTestCase.php

```diff
@@ -153,12 +178,15 @@ public function afterSetup()
         
         if ($this->apiClass) {
             $class = $this->apiClass;
-            $this->api = new $class('api', $this->app);
+            $modelClass = $this->modelClass;
+            $this->api = new $class($modelClass::ngRestApiEndpoint(), $this->app);
         }
         
         if ($this->controllerClass) {
             $class = $this->controllerClass;
-            $this->controller = new $class('controller', $this->app);
+            $className = preg_replace('/^.*\\\/', '', $class::className());
```

Comment:
Maybe there is already a function which does this in: https://luya.io/api/luya-helpers-Inflector otherwise this is good for me

Reply:
Here the basename of the namespaced controller classname is extracted